scio.rb

```diff
@@ -20,7 +20,7 @@ def install
           fi
       done
 
-      CLASSPATH=#{libexec}/"scio-repl-#{version}.jar":$CLASSPATH exec java ${java_opts[*]} ${scio_opts[*]} com.spotify.scio.repl.ScioShell
+      CLASSPATH=#{libexec}/"scio-repl-#{version}.jar"${CLASSPATH:+:${CLASSPATH}} exec java ${java_opts[*]} ${scio_opts[*]} com.spotify.scio.repl.ScioShell
```

Comment:
`#{libexec}/"scio-repl-#{version}.jar"` should be `"#{libexec}/scio-repl-#{version}.jar"` maybe?
Although we're pretty sure no space in the path and quotes are unnecessary?